src/docs/getting-started/probe-setup.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Probe Setup"
+description: "How to set up your debug probe to work with probe-rs."
+order: 20
+---
+
+The probe-rs project supports three probes out of the box.
+
+Most of them require little to no setup. For linux, [udev](#udev-rules) rules
+are required for non admin access.
+
+The [ST-Link](#st-link), to [J-Link](#segger-j-link) and [CMSIS-DAP](#cmsis-dap)
+based probes are supported.
+
+## udev rules
+
+By default, the debug probes are only accessible by users with root privileges
+on Linux based systems. It is recommend to use appropriate udev rules to allow
+users without root privileges access to the debug probes as well.
+
+1. Download the file [69-probe-rs.rules](/files/69-probe-rs.rules)[^1] and place
```

Comment:
The [^1] doesn't seem to work, and the link should download the file, not open it in  a new window.

Reply:
Use something like `<a href=xxx download>69-probe-rs.rules</a>`? Or do fancy javascript frameworks not support this :stuck_out_tongue: ?

Reply:
Oh there is a download hint? Did not know!